internal/gitinterface/commit.go

```diff
@@ -103,7 +103,7 @@ func VerifyCommitSignature(ctx context.Context, commit *object.Commit, key *tuf.
 }
 
 // CreateCommitObject returns a commit object using the specified parameters.
-func CreateCommitObject(gitConfig *config.Config, treeHash plumbing.Hash, parentHash plumbing.Hash, message string, clock clockwork.Clock) *object.Commit {
+func CreateCommitObject(gitConfig *config.Config, treeHash plumbing.Hash, parentHashes []plumbing.Hash, message string, clock clockwork.Clock) *object.Commit {
```

Comment:
This change needs to be incorporated in the common package as well.

Reply:
Fixed